src/components/Dashboard /PartnerData/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import "./styles.scss";
+// import { METHODS } from "../../../services/api";
+import { Link } from "react-router-dom";
+import { PATHS } from "../../../constant";
+import { useSelector } from "react-redux";
+
+function PartnerDashboard() {
+  const [partners, setPartners] = useState([]);
+  const [searchTerm, setSearchTerm] = useState("");
+  const user = useSelector(({ User }) => User);
+
+  useEffect(() => {
+    axios
+      .get(` https://api.merakilearn.org/partners`, {
```

Comment:
the baseUrl should not be hardcoded. Search for `process.env.REACT_APP_MERAKI_URL` in the codebase and see how it is used.

Reply:
Actually, in development API there is no data for the partner,  for testing I used this. I removed now